Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Collections/BaseObjectCollection.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.Collections.Generic;
+using UnityEditor;
```

Comment:
Pretty sure this breaks the build

Reply:
https://dev.azure.com/aipmr/MixedRealityToolkit-Unity-CI/_build?definitionId=15

The automated builds are fine.

Reply:
Fine, don't take my word for it.

Reply:
@StephenHodgson the build from Unity to VS worked for me and @keveleigh said the automated builds were fine. Can you elaborate what build is broken for you?

Reply:
Editor code should never make it into the final build. I'm surprised the build even succeeded.

Must be getting removed by the compiler or something, but really this should not be included.

Reply:
@keveleigh Stephen is right this include will definitely break the build. Something must be funky with the automated builds.

Reply:
@railboy Do you (or anybody else) have a build break or actionable call stack? I have not had issues building after this change.

Looks to me like it's currently not breaking the build because there's something in the namespace that's available at runtime (the ⚠ means only available in the Editor): 
![image](https://user-images.githubusercontent.com/3580640/58040291-036e3180-7aea-11e9-9777-35e5e43e6166.png)

Reply:
Might just be a misconfiguration on Unity's part which we probably shouldn't depend on though.

Reply:
>Do you (or anybody else) have a build break or actionable call stack?

@keveleigh Nope, looks like a non-issue after all.